### ISSUE TITLE
directus-extension build -w --no-minify stops seeing changes

### ISSUE THREAD

**Issue by julianko** [directus/directus#16623]
### Describe the Bug

When this Problem occurs, the process correctly detects changes (according to the terminal) and rebuilds, but the generated file in `dist/` is always the same. Only after restarting the build process will it reflect the changes made to source files.

I'm using node v16.18.0 on Windows 11.

### To Reproduce

1. create a new extension: `npm init directus-extension`. I did module with typescript - not sure if it matters.
2. run `npm run dev` in the extension dir
3. make changes to `module.vue` and watch the generated file in `dist/` change live

I haven't been able to determine what exactly causes the problem, on a completely new module it seems to work correctly, but when I add another vue module and import that in `module.vue`, it starts to break at some point.

### What version of Directus are you using?

9.21.0

### What version of Node.js are you using?

16.18.0

### What database are you using?

SQLite

### What browser are you using?

Brave

### How are you deploying Directus?

local

**Comment by stefaansterk** [comment]
I have noticed the same issue

**Comment by nickrum** [comment]
Duplicate of #16209
